policyengine_us/parameters/gov/states/wv/tax/income/subtractions/social_security_benefit/threshold.yaml

```diff
@@ -0,0 +1,25 @@
+description: West Virginia reduces its status deduction with adjusted federal AGI exceeding this amount.
+
+metadata:
+  period: year
+  unit: currency-USD
+  label: Adjusted federal AGI threshold for WV taxpayers eligible for an status deduction
+  breakdown:
+  - filing_status
+  reference:
+  - title: West Virginia Personal Income Tax Forms And Instructions 2022
+    href: https://tax.wv.gov/Documents/PIT/2022/PersonalIncomeTaxFormsAndInstructions.2022.pdf#Page=25
+  - title: Code of West Virginia §11-21-12 (c)(8)
+    href: https://code.wvlegislature.gov/11-21-12/
+
+
+JOINT:
+  2022-01-01: 100_000
```

Comment:
add 2021 values + 2021 tax form reference

Reply:
also 2020 values as they are in the law

Reply:
need to make a scale "single_amount" parameter to represent the change from 35% in 2020 to 65% in 2021 and 100% in 2022